pkg/cmd/extensions/manager.go

```diff
@@ -99,17 +98,39 @@ func (m *Manager) list(includeMetadata bool) []extensions.Extension {
 			continue
 		}
 		var remoteURL string
+		var updatable bool
 		if gitExe != "" {
-			stdout := bytes.Buffer{}
-			cmd := m.newCommand(gitExe, "--git-dir="+filepath.Join(dir, f.Name(), ".git"), "config", "remote.origin.url")
-			cmd.Stdout = &stdout
-			if err := cmd.Run(); err == nil {
-				remoteURL = strings.TrimSpace(stdout.String())
-			}
+			func() {
```

Comment:
why a self executing closure here?

Reply:
I wanted the behavior of that if one of the git commands failed that it would not continue to the next one and discard any errors. This seemed like the cleanest way to do it. I could have extracted a function and passed the couple of args necessary, but this seemed okay also. I am open to other suggestions though.

Reply:
I'm not opposed to self-executing anonymous funcs in some cases, but often they signal that the wrapped expression should have been its own func instead.